Fix hardcoding of /usr/bin/perl, use perl available through $PATH
Fix hardcoding of /bin/bash, use bash available through $PATH
author: Nathan S. Watson-Haigh (ACPFG)
--- a/scripts/index_bwa.sh
+++ b/scripts/index_bwa.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 source bwa-0.6.1
 
--- a/scripts/nextclip_index_reference.pl
+++ b/scripts/nextclip_index_reference.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 # Script:  nextclip_index_reference.pl
 # Purpose: Produce simple index file of a reference
--- a/scripts/nextclip_lmp_analysis.pl
+++ b/scripts/nextclip_lmp_analysis.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 # Script:  nextclip_lmp_analysis.pl
 # Purpose: NextClip Long Mate Pair analysis pipeline
--- a/scripts/nextclip_make_report.pl
+++ b/scripts/nextclip_make_report.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 # Script:  nextclip_make_report.pl
 # Purpose: Produce PDF report of NextClip analysis
--- a/scripts/nextclip_plot_graphs.sh
+++ b/scripts/nextclip_plot_graphs.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 # Script:  nextclip_plot_graphs.sh
 # Purpose: Plot insert length and read length graphs for NextClip
--- a/scripts/nextclip_sam_parse.pl
+++ b/scripts/nextclip_sam_parse.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 # Script:  nextclip_sam_parse.pl
 # Purpose: Parse SAM files for NextClip
